src/Core/MailTemplates/Handlebars/FamiliesForEnterprise/FamiliesForEnterpriseRemovedFromFamilyUser.text.hbs

```diff
@@ -0,0 +1,6 @@
+{{#>BasicTextLayout}}
+    {{SponsoringOrgName}} has removed the Free Bitwarden Families plan sponsorship.
+    Here’s what that means:
+    Your Free Bitwarden Families sponsorship will charge your stored payment method on {{OfferAcceptanceDate}}. To avoid any disruption in your service, please ensure your payment method on the Subscription page is up to date. Or click the following link: {{{link SubscriptionUrl}}}
```

Comment:
🤔 I don't know if this is intended or not, but using the `link` helper will put in an HTML anchor tag. With this being the plain text form, it might look strange. 

I've seen other [examples](https://github.com/bitwarden/server/blob/main/src/Core/MailTemplates/Handlebars/Auth/FailedTwoFactorAttempts.text.hbs) of `.text.hbs` files that just use `(https://your.url.here)` so if you're using a variable it'd be `({{SubscriptionUrl}})`. If its intended to put the anchor tag there, then feel free to ignore this.